force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -321,18 +396,22 @@ public with sharing class FlowControlLoopsCollections {
      * @return The salary of the employee, or null if the employee is not found in the Map.
      */
     public static Integer getSalary(String employeeName) {
-        // Initialize a Map
-
-        // Add key-value pairs to the Map
-        // add John Doe => 50000
-        // add Jane Smith => 60000
-        // add Sam Brown => 55000
-        // add Alice Johnson => 65000
-
-        // Get the salary of the employee
+        
+        Map<String, Integer> employeeMap = new Map<String, Integer>();
+                
+        employeeMap.put('John Doe', 50000);         // add John Doe => 50000
+        employeeMap.put('Jane Smith', 60000);       // add Jane Smith => 60000
+        employeeMap.put('Sam Brown', 55000);        // add Sam Brown => 55000
+        employeeMap.put('Alice Johnson', 65000);    // add Alice Johnson => 65000
+
+        Integer salary = employeeMap.get(employeeName);     // Get the salary of the employee
+        
+        if(salary == null) {       // Return the salary of the employee, or null if the employee is not found in the Map
+            return null;
+        }                                                
+                                                            
+        return salary;
```

Comment:
Good work

Like before you could use the Map constructor to initialize your values. And you dont need the variable integer if all you are using it for is to return. 

Just return employeeMap.get()